src/modules/menu/scss/topics-image.scss

```diff
@@ -1,49 +1,145 @@
-  (key: nga,           position: -220px -60px),
-  (key: luftbilder,    position: -220px -120px),
-  (key: gewiss,        position: -220px -180px),
-  (key: dev,           position: -220px -240px),
-  (key: cadastre,      position: -220px -300px),
-  (key: meteoschweiz,  position: -220px -360px),
-  (key: isos,          position: -220px -420px),
-  (key: swisstopo,     position: -330px 0),
-  (key: ivs,           position: -330px -60px),
-  (key: wildruhezonen, position: -330px -120px),
-  (key: inspire,       position: -330px -180px),
-  (key: geothermie,    position: -330px -240px),
-  (key: astra,         position: -330px -300px),
-  (key: bfs,           position: -330px -360px),
-  (key: bav,           position: -330px -420px),
+    (
+        key: blw,
+        position: 0 0,
+    ),
+    (
+        key: kgs,
+        position: 0 -60px,
+    ),
+    (
+        key: sachplan,
+        position: 0 -120px,
+    ),
+    (
+        key: vu,
+        position: 0 -180px,
+    ),
+    (
+        key: ech,
+        position: 0 -240px,
+    ),
+    (
+        key: schneesport,
+        position: 0 -300px,
+    ),
+    (
+        key: emapis,
+        position: 0 -360px,
+    ),
+    (
+        key: notruf,
+        position: 0 -420px,
+    ),
+    (
+        key: are,
+        position: -110px 0,
+    ),
+    (
+        key: funksender,
+        position: -110px -60px,
+    ),
+    (
+        key: geol,
+        position: -110px -120px,
+    ),
+    (
+        key: aviation,
+        position: -110px -180px,
+    ),
+    (
+        key: verteidigung,
+        position: -110px -240px,
+    ),
+    (
+        key: energie,
+        position: -110px -300px,
+    ),
+    (
+        key: geodesy,
+        position: -110px -360px,
+    ),
+    (
+        key: schule,
+        position: -110px -420px,
+    ),
+    (
+        key: bafu,
+        position: -220px 0,
+    ),
+    (
+        key: nga,
+        position: -220px -60px,
+    ),
+    (
+        key: luftbilder,
+        position: -220px -120px,
+    ),
+    (
+        key: gewiss,
+        position: -220px -180px,
+    ),
+    (
+        key: dev,
+        position: -220px -240px,
+    ),
+    (
+        key: cadastre,
+        position: -220px -300px,
+    ),
+    (
+        key: meteoschweiz,
+        position: -220px -360px,
+    ),
+    (
+        key: isos,
+        position: -220px -420px,
+    ),
+    (
+        key: swisstopo,
+        position: -330px 0,
+    ),
+    (
+        key: ivs,
+        position: -330px -60px,
+    ),
+    (
+        key: wildruhezonen,
+        position: -330px -120px,
+    ),
+    (
+        key: inspire,
+        position: -330px -180px,
+    ),
+    (
+        key: geothermie,
+        position: -330px -240px,
+    ),
+    (
+        key: astra,
+        position: -330px -300px,
+    ),
+    (
+        key: bfs,
+        position: -330px -360px,
+    ),
+    (
+        key: bav,
+        position: -330px -420px,
+    )
```

Comment:
this block I kind of dislike post format, less readability...
Could you wrap it around some "no format" comments or something that deactivate format for these lines?